src/extension/sldService/src/test/java/org/geoserver/sldservice/rest/ClassifierTest.java

```diff
@@ -23,37 +23,42 @@
 import org.geotools.filter.function.FilterFunction_parseDouble;
 import org.geotools.styling.PointSymbolizer;
 import org.geotools.styling.Rule;
-import org.junit.Before;
 import org.junit.Test;
 import org.springframework.mock.web.MockHttpServletResponse;
 import org.w3c.dom.Document;
 
-public class ClassifierTestSupport extends SLDServiceBaseTest {
+public class ClassifierTest extends SLDServiceBaseTest {
```

Comment:
This test is a true unit one, does not need to be a GeoServerSystemTest subclass

Reply:
Also, name must end with "Test", otherwise maven would not run this class (this is the reason why removing the XOM dependency did not make any difference)